CHANGELOG.md

```diff
@@ -7,6 +7,17 @@
 * More customization - allowlisting and denylisting of error codes and scripts
 * And more...
 
+## v2.4.0
+
+* Support for `ash`, `dash`, `ksh` and `bats` shell interpreters
+* Improve autodetection of shell scripts
+  * Support for detection based on shellcheck directive
+  * Support for generally used shebangs like: `#!/usr/bin`, `#!/usr/local/bin`, `#!/bin/env␣`, `#!/usr/bin/env␣` and `#!/usr/local/bin/env␣`
+
+## v2.3.6
+
+* Fix tool name in SARIF reports
```

Comment:
This should not be part of this PR, right? (May you forgot to do a rebase.)

Reply:
This is correct, CHANGELOG entry was missing in last PR so I added here